doc/acl/acl_stage_capability.md

```diff
@@ -0,0 +1,220 @@
+# Egress mirroring support and ACL action capability check
+
+# Table of Contents
+
+#### Revision
+| Rev |  Date   |       Author       | Change Description |
+|:---:|:-------:|:------------------:|:------------------:|
+| 0.1 | 2019-05 |   Blyschak Stepan  | Initial Version    |
+
+## Motivation
+Not all ASICs support all actions on ingress, egress stages
+
+E.g.: Egress mirror action on ingress stage or vice versa might be not supported
+
+## Design
+
+## 1. Egress mirroring support
+
+The proposed new schema:
+
+### ACL_RULE_TABLE
+```
+mirror_action = 1*255VCHAR                         ; refer to the mirror session (implicitly ingress for backward compatibility)
+mirror_ingress_action = 1*255VCHAR                 ; refer to the mirror session
+mirror_egress_action = 1*255VCHAR                  ; refer to the mirror session
+```
+
+e.g.:
+```
+{
+  "ACL_RULE": {
+        "EVERFLOW|RULE_1": {
+            "MIRROR_EGRESS_ACTION": "everflow0",
```

Comment:
each acl table has binding point, egress or ingress. 

Are we allowing ingress acl to have egress mirror action? or we should restrict only egress acl have egress mirror action, meanwhile, ingress acl can have ingress mirror action?

Reply:
@lguohan 
There is not restriction. EVERFLOW table in example can be either ingress stage or egress stage.
Application (acl-loader) should look  in SWITCH_CAPABILITY table to check if action is supported on specific acl table stage.
The example below in the design shows the capability table example on Mellanox.
So you can see that on Mellanox egress mirror on ingress acl is not supported.